content/upcoming/announcing_babelfish.md

```diff
@@ -0,0 +1,82 @@
+---
+author: abeaumont
+date: 2017-06-06
+title: "Announcing Babelfish"
+draft: true
+image: **TODO: Babelfish image (/post/announcing-babelfish/babelfish.jpg)**
+description: "Announcing Babelfish, the project we are developing to build
+representations of source code."
+categories: ["technical"]
+---
+
+At source{d} we believe there's a possibility for programs to write their own
```

Comment:
I took that directly from [Our Roadmap blogpost](https://blog.sourced.tech/post/our-roadmap/): _At source{d} we believe that neural networks have opened the possibility for programs to write their own code and for new forms of automatic programming to emerge._
It may be better to leave the whole phrase to have a more complete context?
`At source{d} we believe there's a possibility for programs to write their own code and for new forms of automatic programming to emerge.`